src/Tempest/Auth/src/CanAuthorize.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth;
+
+use UnitEnum;
+
+interface CanAuthorize
+{
+    public function grantPermission(string|UnitEnum $permission): self;
```

Comment:
Should the Authorizer really have this?

Reply:
Not sure yet, I'm in the sketching phase right now :)